templates/CRUD/add_rows.html

```diff
@@ -6,11 +6,13 @@
         {% for field in form %}
             <p>
                 {{ field.label_tag }}
-                {{ field }}
-
                 {% if field.help_text %}
-                    {% comment %} <p class="help_text"> {{ field.help_text }} </p> {% endcomment %}
+                    <button class="help_text" title= " {{ field.help_text }} ">?</button>
```

Comment:
Looks like your trying to create a tooltip here with a button and hover- here's the docs for that: https://materializecss.com/tooltips.html. No need to initialize it-- all the components have been initialized.